_sources/non-integral/1.0.0.0/revisions/2.cabal

```
@@ -0,0 +1,42 @@
+cabal-version:      3.0
+name:               non-integral
+version:            1.0.0.0
+license:            Apache-2.0
+maintainer:         operations@iohk.io
+author:             IOHK
+description:        Implementation decision for non-integer calculations
+build-type:         Simple
+extra-source-files:
+    README.md
+    CHANGELOG.md
+
+source-repository head
+    type:     git
+    location: https://github.com/input-output-hk/cardano-ledger.git
+    subdir:   libs/non-integral
+
+library
+    exposed-modules:  Cardano.Ledger.NonIntegral
+    hs-source-dirs:   src
+    default-language: Haskell2010
+    ghc-options:
+        -Wall -Wcompat -Wincomplete-record-updates
+        -Wincomplete-uni-patterns -Wredundant-constraints -Wunused-packages
+
+    build-depends:    base >=4.14 && <4.20
```

You've set it on master to:
```suggestion
    build-depends:    base >=4.14 && <5
```

Just to avoid confusion in the future it would be nice to have this revision reflect the same state as on master